src/Components/Details/Details.js

```diff
@@ -40,33 +43,41 @@ const DetailsModal = ({ title, determineFav, toggleFav, isFavorite }) => {
   
   return (
     <>
-      <section className="details_main" data-cy="modal">
-        <article className="details_box left">
-          <img data-cy="album-cover" className="details_album-cover shadow" src={data1.album.coverImage} alt={`${data1.album.title} album cover`}/>
-          <div className="details_links">
-            <IconButton
-              className="details_favorite-button click" data-cy="favorites-button"
-              aria-label={"Add to Favorites"}
-              onClick={ () => handleFavoriteClick() }
-              >
-              { determineFav(isFav) }
-            </IconButton>
-            <a data-cy="details_discogs-link" className="details_discogs-link" href={data1.album.uri} target="_blnk">
-              <img className="details_discogs-logo" src={discogsLogo} alt="discogs logo"/>
-            </a>
-          </div>
-          <div className="details_main-text">
-            <p data-cy="artist-name">Artist: {data1.album.artists[0].name}</p>
-            <p data-cy="album-title">Album: {data1.album.title}</p>
-            <p data-cy="release-year">Released: {data1.album.year}</p>
-            <div className='genre-container' data-cy='genre-container'>
-              { displayGenres(data1.album.genres) }
+      <section className="details_main" data-cy="modal"> 
+        <Link to="/" className="details_back-button click">
+          <KeyboardReturnIcon className="details_return-arrow"/>
+            Back
+        </Link> 
+        <div className="details_content">
+          <article className="details_box left">
+            <img data-cy="album-cover" className="details_album-cover shadow" src={data1.album.coverImage} alt={`${data1.album.title} album cover`}/>
```

Comment:
I think class names should come first and data-cy second

Reply:
Thanks for letting me know! This change has now been made.